src/rules/let_in.rs

```diff
@@ -94,38 +94,46 @@ pub fn rule(
         }
     }
     steps.push_back(crate::builder::Step::Format(child.element));
-    match layout {
-        crate::config::Layout::Tall => {
-            if indent {
-                steps.push_back(crate::builder::Step::Indent);
-            }
-        }
-        crate::config::Layout::Wide => {}
-    }
```

Comment:
This causes this:

```diff
--- a/lib/cli.nix
+++ b/lib/cli.nix
@@ -82,5 +82,5 @@ rec {
           then mkList k v
           else mkOption k v;
       in
-        builtins.concatLists (lib.mapAttrsToList render options);
+      builtins.concatLists (lib.mapAttrsToList render options);
 }
```

```diff
@@ -22,11 +21,11 @@ rec {
   attrByPath = attrPath: default: e: let
     attr = head attrPath;
   in
-    if attrPath == [ ]
-    then e
-    else if e ? ${attr}
-    then attrByPath (tail attrPath) default e.${attr}
-    else default;
+  if attrPath == [ ]
+  then e
+  else if e ? ${attr}
+  then attrByPath (tail attrPath) default e.${attr}
+  else default;
 ```

the code is missing some look-ahead between the indent and the tokens that come next, so that one can start formatting after the `in` correctly